app/views/users/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>
+  <%= @user.username %>
+</h1>
+
+<h2>Own photos</h2>
+
+<% @user.own_photos.each do |photo| %>
+  <div class="row mb-4">
+    <div class="col-md-6 offset-md-3">
+      <div class="card">
+        <%= image_tag photo.image, class: "card-img-top" %>
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```

Comment:
Maybe consider commenting this `img` out for now as it's just a placeholder

Reply:
Good catch! I think we will be able to replace it a bit later, so I kept it as it is.

Reply:
Yes great catch, usually if an image tag is empty, its good either remove it, comment it out, or add default values to it :)